tests/ansible_lint_rules/NoFormattingInWhenRule.py

```diff
@@ -0,0 +1,32 @@
+from ansiblelint import AnsibleLintRule
```

Comment:
Let's add some comments to this file for the functions so that folks new to ansible-lint can dive in more easily.